packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -71,9 +194,34 @@ const BoldChunk = (chunks) => <Typography fontWeight="bold">{chunks}</Typography
  * SelectedEntriesModalContent
  * -----------------------------------------------------------------------------------------------*/
 
-const SelectedEntriesModalContent = ({ onToggle, onConfirm }) => {
+const SelectedEntriesModalContent = ({ onToggle, onConfirm, onRefreshData }) => {
   const { formatMessage } = useIntl();
-  const { selectedEntries } = useTableContext();
+  const { selectedEntries, rows, isLoading } = useTableContext();
+  const { contentType, components } = useSelector(listViewDomain());
+
+  /**
+   * @returns {{validIds: number[], errors: Object.<number, string>}} - Returns an object with the valid ids and the errors
+   */
+  const validateEntriesToPublish = React.useCallback(() => {
+    const validations = { validIds: [], errors: {} };
+    // Create the validation schema based on the contentType
+    const schema = createYupSchema(contentType, { components }, { isDraft: false });
+    // Validate each entry
+    rows.forEach((entry) => {
+      try {
+        schema.validateSync(entry, { abortEarly: false });
+        validations.validIds.push(entry.id);
+      } catch (e) {
+        validations.errors[entry.id] = getYupInnerErrors(e);
+      }
+    });
+
+    return validations;
+  }, [rows, contentType, components]);
+
+  const { errors } = validateEntriesToPublish();
+
+  const atLeastOneEntryHasNoErrors = selectedEntries.some((entryId) => !errors[entryId]);
 
   return (
     <ModalLayout onClose={onToggle} labelledBy="title">
```

Comment:
I think you will need to update the header to include the validations:

> 5 Entries ready to publish - 2 entries waiting for action

https://www.figma.com/file/64cWvO8k1TWBDz39Kz1pZt/%5B%F0%9F%8F%97WIP%5D-Content-Studio---Bulk-publish?type=design&node-id=309-15534&mode=design&t=rb9UpzqNaPzgcI2W-0